src/composables/RoutingSlip/useCreateRoutingSlip.ts

```diff
@@ -42,6 +59,17 @@ export function useCreateRoutingSlip (_, context) {
     modalDialogRef.value.open()
   }
 
+  // Display success
+  function displaySuccessNotification () {
+    // Update modal dialog props and display
+    modalDialogTitle.value = i18n.t('createRoutingSlipSuccessTitle').toString()
```

Comment:
can we make all these refs as reactive one object

Reply:
yup. changed